src/app-layer-ssl.c

```diff
@@ -1527,6 +1538,7 @@ static int SSLv3ParseHandshakeType(SSLState *ssl_state, const uint8_t *input,
             rc = SSLv3ParseHandshakeTypeCertificate(ssl_state,
                     direction ? &ssl_state->server_connp : &ssl_state->client_connp, initial_input,
                     input_len);
+            DEBUG_VALIDATE_BUG_ON(ssl_state->client_connp.cert0_sans_len != 0);
```

Comment:
Why this debug assertion ?

Reply:
Because I could only find SAN being used in the toclient dir, read that the certs may be present in both dirs in case of mTLS. So, wasn't sure if SAN would be there on both directions. Was hoping our QA lab would trigger this assertion if it should be possible in both dirs..

Reply:
Maybe the use case is only in this direction, but you can have certificates both ways, so I would remove this debug assertion

Reply:
ok. thank you